keras_contrib/layers/normalization.py

```diff
@@ -373,4 +374,186 @@ def get_config(self):
         base_config = super(BatchRenormalization, self).get_config()
         return dict(list(base_config.items()) + list(config.items()))
 
+
 get_custom_objects().update({'BatchRenormalization': BatchRenormalization})
+
+
+class GroupNormalization(Layer):
+    """Group normalization layer
+
+    Group Normalization divides the channels into groups and computes within each group
+    the mean and variance for normalization. GN's computation is independent of batch sizes,
+    and its accuracy is stable in a wide range of batch sizes
+
+    # Arguments
+        groups: Integer, the number of groups for Group Normalization.
```

Comment:
Can this include advice on how to choose the integer?

Reply:
Hmm, I can mention that setting groups to 1 makes this identical to Layer Normalization, and setting it to the number of input channels makes this Instance Normalization?

Reply:
Potential fix, use the private `_moments` method directly from the CNTK backend. It will be more fragile to internal Keras changes, but I think moments op is too general to have any major changes.